zio-http/src/test/scala/zhttp/service/ClientSpec.scala

```diff
@@ -11,6 +13,33 @@ object ClientSpec extends HttpRunnableSpec(8082) {
     testM("respond Ok") {
       val actual = Client.request("http://api.github.com/users/zio/repos", ClientSSLOptions.DefaultSSL)
       assertM(actual)(anything)
-    },
+    } +
+      testM("non empty content") {
+        val actual          = Client.request("https://api.github.com/users/zio", ClientSSLOptions.DefaultSSL)
+        val responseContent = actual.flatMap(_.getBody)
+        assertM(responseContent)(isNonEmpty)
+      }
+      +
+      testM("POST request expect non empty response content") {
+        val url             = Task.fromEither(URL.fromString("https://api.github.com/users/zio"))
+        val endpoint        = url.map(u => Method.POST -> u)
+        val headers         = List(Header.userAgent("zio-http test"))
+        val response        = endpoint.flatMap { e =>
+          Client.request(e, headers, HttpData.fromText("test"))
+        }
+        val responseContent = response.flatMap(_.getBody)
+        assertM(responseContent)(isNonEmpty)
+      }
+      +
+      testM("empty content") {
+        val actual          = Client.request("http://api.github.com/users/zio/repos", ClientSSLOptions.DefaultSSL)
+        val responseContent = actual.flatMap(_.getBody)
+        assertM(responseContent)(isEmpty)
+      } +
+      testM("text content") {
+        val actual          = Client.request("https://www.google.com", ClientSSLOptions.DefaultSSL)
+        val responseContent = actual.flatMap(_.getBodyAsString)
+        assertM(responseContent)(containsString("Google"))
+      },
```

Comment:
The problem with these tests is that — I can't make it work offline. It's not a major problem, however, I feel that tests become less flaky when they don't depend on external envs. 

Unless really required we should use our own server and client for all tests.